Procfile

```diff
@@ -1,3 +1,3 @@
-web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development} -C puma_config.rb
```

Comment:
What do you think about moving all the puma related configurations to the puma_config.rb file? Right now, we have some command line configs, and production has the `WEB_CONCURRENCY` environment variable. They’re a bit scattered everywhere.

```ruby
environment ENV["RACK_ENV"] || "development"
port ENV["PORT"] || "3000"
threads 5, 5
workers ENV["WEB_CONCURRENCY"] || 0
preload_app!
```

Reply:
That makes sense to me.